src/caffe/proto/caffe.proto

```diff
@@ -280,6 +280,10 @@ message LayerParameter {
 
   // The blobs containing the numeric parameters of the layer.
   repeated BlobProto blobs = 7;
+  
+  // Specifies on which bottoms the backpropagation should be skipped.
+  // The size must be either 0 or equals to the number of bottoms.
+  repeated bool skip_propagate_down = 11;
```

Comment:
I know I suggested this name (sorry), but now I think I was wrong and the field name should just be `propagate_down` (thereby inverting the values it takes now) and it should default to the net's inferred settings if unspecified. (`propagate_down` is consistent with the current code and "`false` means skip" is more intuitive than "`true` means skip" to me.)